### PR TITLE
Added Slovenian translation

### PR DESCRIPTION
**Pull request by thehijacker**
Great project and I wish to contribute. Here are Slovenian translations. I kindly ask to include them in next version.

### CODE REVIEW

**Comment by ellite** [comment]
Hi, thank you very much for the contribution. 

Could you please have a look at https://github.com/ellite/Wallos?tab=readme-ov-file#translations as there are some parts missing?
Thank you again.

File: includes/i18n/sl.php [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+    'file_type_error' => "Vrsta datoteke ni podprta.",
+    'user_details' => "Podrobnosti o uporabniku",
+    "monthly_budget" => "Mesečni proračun",
+    "budget_info" => "Mesečni proračun se uporablja za izračun statistike",
+    "household" => "Gospodinjstvo",
+    "save_member" => "Shrani člana",
+    "delete_member" => "Izbriši člana",
+    "cant_delete_member" => "Ne morem izbrisati glavnega člana",
+    "cant_delete_member_in_use" => "Ne morem izbrisati člana, ki je v uporabi v naročnini",
+    'household_info' => "E-poštno polje omogoča članom gospodinjstva, da so obveščeni o naročninah, ki bodo potekle."
```
Comment (ellite) [inline includes/i18n/sl.php:126]: There's a comma , missing on this line.

Comment (thehijacker) [reply includes/i18n/sl.php]: Sorry. I think I have everything now.

**Comment by ellite** [comment]
Thank you very much. I'll merge it after work.

**Comment by ellite** [comment]
Sorry again, this part is still missing:

Add your language code to includes/i18n/languages.php in the format "en" => "English". Please use the original language name and not the english translation.


Would be awesome if you could add it as well.
Thanks :)

**Comment by thehijacker** [comment]
Here we go. Sorry. Missed it.

**Comment by ellite** [comment]
Merged, thanks.
Will go with next release soon.

**Pull request merged**